src/pinetree/feature.cpp

```diff
@@ -133,7 +133,7 @@ Mask::Mask(int start, int stop,
 /**
  * TODO: refactor to make vector rather than map
  */
-bool Mask::CheckInteraction(const std::string &name) {
+bool Mask::CheckInteraction(const std::string &name) const{
```

Comment:
Should there be a space between `const` and `{`? Are we using any specific style guide for the code?

Reply:
Yes it looks like there should be a space here. I don't think we are using a specific style guide - I haven't seen one mentioned anywhere. Actually, pretty sure we don't have developer guidelines for this project - maybe that's something that should be added to the docs.